modules/tinymce/src/plugins/template/main/ts/api/Commands.ts

```diff
@@ -7,10 +7,20 @@
 
 import { Fun } from '@ephox/katamari';
 
+import Editor from 'tinymce/core/api/Editor';
+
 import * as Templates from '../core/Templates';
+import * as Dialog from '../ui/Dialog';
+
+const showDialog = (editor: Editor) => {
+  return (templates) => {
+    Dialog.open(editor, templates);
+  };
+};
 
 const register = (editor) => {
   editor.addCommand('mceInsertTemplate', Fun.curry(Templates.insertTemplate, editor));
+  editor.addCommand('mceCreateTemplateList', Templates.createTemplateList(editor, showDialog(editor)));
```

Comment:
This doesn't match naming conventions and is inconsistent with other plugin commands. It should be `mceTemplate` in this case, so can we please rename that?

- [ ] Rename command to keep it consistent

Reply:
Changed to `mceTemplate` but couldn't figure out the convention. Is it mce<PLUGIN_FOLDER> capitalized?

Reply:
Yeah, the convention is normally just `mce<PLUGIN NAME>` where the plugin name uses PascalCase. There are some older things that don't follow that, such as `mceCodeEditor`, but we can't change them unfortunately.